src/controllers/getRecipeByIdController.ts

```diff
@@ -0,0 +1,61 @@
+import { Request, Response } from "express";
+import getDatabase from "../Services/databaseConnector";
+import { userIdExists, recipeIdExists } from "../Services/validators";
+import { Connection } from "promise-mysql";
+
+interface Ingredient {id: number, name: string}
```

Comment:
```suggestion
interface Ingredient {
  id: number, 
  name: string
}

```

This is easier to read for interfaces. Also one blank line between.

Unless this is something Prettier has done.